### PR TITLE
[SC-619] Hardhat ethers tests migration

### PR DESCRIPTION
**Pull request by typicalbuster**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,31 +7,29 @@
...
     "devDependencies": {
-        "@1inch/solidity-utils": "2.0.23",
-        "@nomiclabs/hardhat-ethers": "2.1.0",
-        "@nomiclabs/hardhat-etherscan": "3.1.0",
-        "@nomiclabs/hardhat-truffle5": "2.0.6",
-        "@nomiclabs/hardhat-web3": "2.0.0",
-        "@openzeppelin/test-helpers": "0.5.15",
+        "@1inch/solidity-utils": "^2.2.2",
+        "@nomicfoundation/hardhat-chai-matchers": "^1.0.4",
+        "@nomicfoundation/hardhat-network-helpers": "^1.0.6",
```
Comment (zZoMROT) [inline package.json:13]: It looks like a redundant dependency

File: test/ProdTest.js [hunk 2]
```diff
@@ -17,10 +15,11 @@ const multiWrapper = '0x931e32b6d112f7be74b16f7fbc77d491b30fe18c';
 
 describe.skip('ProdTest', async function () {
     before(async function () {
-        // this.compoundWrapper = await CompoundWrapper.new();
+        // this.compoundWrapper = await CompoundWrapper.deploy();
```
Comment (zZoMROT) [inline test/ProdTest.js:18]: I think it will be useful to add `await this.compoundWrapper.deployed();` to the end of this comments

**Review by zZoMROT — APPROVED** [review]

File: test/CurveOracle.js [hunk 3]
```diff
@@ -81,8 +94,9 @@ describe('CurveOracle doesn\'t ruin rates', async function () {
             ],
             tokens.WETH,
         );
+        await this.oldOffchainOracle.deployed();
 
-        this.newOffchainOracle = await OffchainOracle.new(
+        this.deployOffchainOracle = await OffchainOracle.deploy(
```
[the remark this answers is not in the document]
Comment (typicalbuster) [reply test/CurveOracle.js:99]: ?

[the remark this answers is not in the document]
Comment (typicalbuster) [reply test/CurveOracle.js:99]: Good find. Seems like was replaced with .new -> .deploy replace

**Review by zZoMROT — APPROVED** [review]

**Pull request merged**